src/screens/QuestionScreen.tsx

```diff
@@ -79,90 +80,108 @@ function getElement(questionOption: QuestionOption) {
-              } as Answer;
-            } else {
-              questionOption.answer = undefined;
-            }
-          }}
-        />
+        <>
+          <QuestionTitle text={'Video vraag'}></QuestionTitle>
+          <VideoSelector
+            value={getMediaURI(questionOption)}
+            onVideoSelected={function (videoPath: FileSelectedData | undefined): void {
+              if (videoPath) {
+                questionOption.answer = {
+                  id: answerId,
+                  values: [videoPath],
+                } as Answer;
+              } else {
+                questionOption.answer = undefined;
+              }
+            }}
+          />
+        </>
       );
     case QuestionOptionType.VOICE:
       return (
-        <AudioRecorder
-          value={getMediaURI(questionOption)}
-          onAudioRecorded={(audio: FileSelectedData | null) => {
-            if (audio) {
-              questionOption.answer = {
-                id: answerId,
-                values: [audio],
-              } as Answer;
-            } else {
-              questionOption.answer = undefined;
-            }
-          }}
-        />
+        <>
+          <QuestionTitle text={'Audio vraag'}></QuestionTitle>
+          <AudioRecorder
+            value={getMediaURI(questionOption)}
+            onAudioRecorded={(audio: FileSelectedData | null) => {
+              if (audio) {
+                questionOption.answer = {
+                  id: answerId,
+                  values: [audio],
+                } as Answer;
+              } else {
+                questionOption.answer = undefined;
+              }
+            }}
+          />
+        </>
       );
     case QuestionOptionType.MULTIPLE_CHOICE:
       return (
-        <MultipleChoiceList
-          values={questionOption.answer?.values}
-          questionOption={questionOption}
-          onClicked={(values: string[]) => {
-            questionOption.answer = {
-              id: answerId,
-              values: values,
-            } as Answer;
-          }}
-        />
+        <>
+          <QuestionTitle text={'Meerkeuze vraag'}></QuestionTitle>
+          <MultipleChoiceList
+            values={questionOption.answer?.values}
+            questionOption={questionOption}
+            onClicked={(values: string[]) => {
+              questionOption.answer = {
+                id: answerId,
+                values: values,
+              } as Answer;
+            }}
+          />
+        </>
       );
     case QuestionOptionType.RANGE:
       return (
-        <RangeSlider
-          value={questionOption.answer?.values?.[0]}
-          questionOption={questionOption}
-          onChange={(value: number) => {
-            questionOption.answer = {
-              id: answerId,
-              values: [value],
-            } as Answer;
-          }}
-        />
+        <>
+          <QuestionTitle text={'Slider vraag'}></QuestionTitle>
```

Comment:
```suggestion
          <QuestionTitle text={'Schaal'}></QuestionTitle>
```

En accessibility label met "Minimaal X, maximaal Y"